src/xmipp/applications/scripts/tomogram_reconstruction/tomogram_reconstruction.py

```diff
@@ -130,68 +135,80 @@ def defineParams(self):
         self.addParamsLine(' [--tvlambda <tvlambda>]              : Multiplier for TV. Default: 50 for SART-TV. 20 for FISTA.')
         self.addParamsLine(' [--alpha_red <alpha_red>]            : Defines the reduction rate of the TV hyperparameter')
         self.addParamsLine(' [--ratio <ratio>]                    : The maximum allowed image/TV update ration. If the TV update changes the image more than this, the parameter will be reduced. Default is 0.95')
+        self.addParamsLine(' [--backprojector <backprojector>]    : This is the used algorithm for backprojecting at each iteration. ')
         
         self.addParamsLine(' [--blocksize <blocksize>]            : (Used by ossart) Sets the projection block size used simultaneously. If BlockSize = 1 OS-SART becomes SART and if  BlockSize = size(angles,2) then OS-SART becomes SIRT. Default is 20.')
         
         self.addParamsLine(' [--astra]                            : Use the astra library')
         self.addParamsLine(' [--method <method>]                  : List of reconstruction algorithms. See the full list in the description.')
 
         self.addParamsLine(' [--filter <filterToApply> ]          : List of filters for FDK or FBP: *ram_lak  (default). * shepp_logan. * cosine. * hamming. *hann. The choice of filter will modify the noise and some discreatization errors, depending on which is chosen.')
+        self.addParamsLine(' [--normalize <normalizeTi>]          : Preprocess the set of images to present zero mean and unit standard deviation')
         self.addParamsLine(' -o <fnOut>                           : Output filename of the tomogram.')
         self.addParamsLine(' --gpu <gpuId>                        : GPU Ids to be use in the image processing') 
```

Comment:
This parameter definition implies 1..n GPUs can be used, however más adelante it seems that you parse one